joomla/joomla-cms
# [4.0] Fix the web components

Pull Request for Issue # .

### Summary of Changes
- The method `HTMLHelper::webcomponent()` is a little different than the script or stylesheet. This PR aligns it (basically the first param was an array and now is just the script path to be included).

- This also assures that web components won't get loaded multiple times!

### Testing Instructions
Apply patch, go to the backend and observe that alerts, switcher, media field and user field are still working!


### Expected result



### Actual result



### Documentation Changes Required
Yes, a note here, using the same params as script and stylesheet is a good thing and also helpful for all devs since they don't have to remember that this method is weird and needs something different.


@wilsonge this is essential, I've got it wrong here 😓
